addons/mrp/report/report_stock_forecasted.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class ReplenishmentReport(models.AbstractModel):
+    _inherit = 'report.stock.report_product_product_replenishment'
+
+    def _move_draft_domain(self, product_template_ids, product_variant_ids, wh_location_ids):
```

Comment:
shouldn't it be move_domain?

Reply:
Ok pigé

Reply:
Nope, it's to avoid to take the draft MO's moves in count when computing the pending ingoing/outgoing qty.